fixtures/tsconfig/cases/outdir_exclude/tsconfig.json

```
@@ -0,0 +1,6 @@
+{
+  "include": ["**/*.ts"],
```

```suggestion
  "include": ["**/*.ts", "**/*.js", "**/*.d.ts"],
```
`js` and `d.ts` must be included to test `outDir`.